diff --git a/node_modules/react-native-sheets-bottom/examples/actualComponents/Panel.js b/node_modules/react-native-sheets-bottom/examples/actualComponents/Panel.js
index eef1c14..f893197 100644
--- a/node_modules/react-native-sheets-bottom/examples/actualComponents/Panel.js
+++ b/node_modules/react-native-sheets-bottom/examples/actualComponents/Panel.js
@@ -171,7 +171,7 @@ class SwipeablePanel extends Component {
     } = this.props;
 
     return showComponent ? (
-      <Animated.View style={[styles.background]}>
+      <Animated.View style={[styles.background,{height:this.state.status == STATUS.SMALL?this.state.panelSmallHeight:this.state.panelLargeHeight}]}>
         <Animated.View
           style={[
             styles.background,
diff --git a/node_modules/react-native-sheets-bottom/examples/actualComponents/Styles.js b/node_modules/react-native-sheets-bottom/examples/actualComponents/Styles.js
index 88efb76..fbeb3e5 100644
--- a/node_modules/react-native-sheets-bottom/examples/actualComponents/Styles.js
+++ b/node_modules/react-native-sheets-bottom/examples/actualComponents/Styles.js
@@ -6,11 +6,12 @@ export const FULL_WIDTH = Dimensions.get('window').width;
 export default StyleSheet.create({
   background: {
     alignItems: 'center',
-    height: FULL_HEIGHT,
     justifyContent: 'center',
     position: 'absolute',
     width: FULL_WIDTH,
     zIndex: 1,
+    bottom:0,
+
   },
   panel: {
     backgroundColor: 'white',
